x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -22,6 +22,7 @@ export const FIND_CSP_RULE_TEMPLATE_ROUTE_PATH = '/internal/cloud_security_postu
 export const FIND_CSP_RULE_TEMPLATE_API_CURRENT_VERSION = '1';
 
 export const DETECTION_RULE_ALERTS_STATUS_API_CURRENT_VERSION = '1';
+export const DETECTION_RULE_RULES_API_CURRENT_VERSION = '2023-10-31';
```

Comment:
Hey @opauloh, this is a public API? 
Please note that internal APIs should use single, larger-than-zero major version. Ex. 1 /2 /3 while public endpoints using dates. 

[versioning-http-apis](https://docs.elastic.dev/kibana-dev-docs/versioning-http-apis#public-endpoints)